plugin/db/mysql/restore.go

```diff
@@ -349,6 +349,83 @@ func (driver *Driver) GetLatestBackupBeforeOrEqualTs(ctx context.Context, backup
 	return driver.getLatestBackupBeforeOrEqualBinlogCoord(validBackupList, *targetBinlogCoordinate, mode)
 }
 
+func (driver *Driver) syncBinlogMetaFileFromCloud(ctx context.Context, downloader *bbs3.Client) error {
```

Comment:
nit: most important information comes first meaning that the caller should be defined above callee.